Python/minestat/__init__.py

```diff
@@ -797,12 +797,13 @@ def __parse_json_payload(self, payload_raw: Union[bytes, bytearray]) -> ConnStat
       self.motd = json.dumps(payload_obj["description"])
     self.stripped_motd = self.motd_strip_formatting(payload_obj.get("description", ""))
 
-    self.max_players = payload_obj["players"]["max"]
-    self.current_players = payload_obj["players"]["online"]
+    players = payload_obj.get("players", {"max":-1, "online":-1})
+    self.max_players = players["max"]
+    self.current_players = players["online"]
 
     # There may be a "sample" field in the "players" object that contains a sample list of online players
-    if "sample" in payload_obj["players"]:
-      self.player_list = [player["name"] for player in payload_obj["players"]["sample"]]
+    if "sample" in players:
+      self.player_list = [player["name"] for player in players["sample"]]
```

Comment:
I think this only works if the whole "players" object is missing. If either the "max" or "online" properties are missing but the players element exists, it is still going to raise an exception here. I would prefer if that would also be cought.